autofill/autofill-impl/src/main/res/drawable/ic_check_recolorable_24.xml

```diff
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:pathData="M12,22C17.523,22 22,17.523 22,12C22,6.477 17.523,2 12,2C6.477,2 2,6.477 2,12C2,17.523 6.477,22 12,22Z"
+      android:fillColor="#21C000"
+      android:fillType="evenOdd"/>
+  <path
+      android:pathData="M16.6,8.28C16.998,8.611 17.052,9.202 16.72,9.6L11.72,15.6C11.552,15.802 11.306,15.925 11.042,15.937C10.779,15.948 10.523,15.849 10.337,15.663L7.337,12.663C6.971,12.297 6.971,11.703 7.337,11.337C7.703,10.971 8.297,10.971 8.663,11.337L10.937,13.611L15.28,8.4C15.611,8.002 16.202,7.948 16.6,8.28Z"
+      android:fillColor="#ffffff"
```

Comment:
unsure if applies, but best check how this works in both themes, and maybe refer to the actual color.

Reply:
Working as intended in both themes